core/src/test/java/org/jsmart/zerocode/core/utils/TokenUtilsTest.java

```diff
@@ -94,12 +93,41 @@ public void testFixedRandomTokenReplace() {
     }
 
     @Test
-    public void testFixedRandomUniqueness() {
+    public void testFixedLengthRandomNumberUniqueness() {
         String result = resolveKnownTokens("${RANDOM.NUMBER:12},${RANDOM.NUMBER:12}");
         String[] split = result.split(",");
-        assertTrue(split[0] != split[1]);
```

Comment:
this line was clearly buggy here, comparing strings with == instead of .equals() method